app/views/catalog/_arclight_index_default.html.erb

```diff
@@ -17,7 +17,7 @@
   </div>
 
   <div class="col pl-0 d-flex flex-wrap" data-has-containers="<%= document.containers.present? %>">
-    <div class="documentHeader order-0" data-document-id="<%= document.id %>">
+    <div class="documentHeader order-0" data-document-id="<%= document.id %>" data-repository-id="<%= document&.repository_config&.slug %>">
```

Comment:
I was discussing recently with @estelendur about having to dig through the "config" to get the slug. It seemed we probably want to get right with Demeter and put a method to get repository ID on the doc. Maybe now, maybe later.

Reply:
Added `repository_id` to the document class.